internal/acctest/config/captchaproviders/settings/captcha_provider_settings_resource_gen_test.go

```diff
@@ -0,0 +1,67 @@
+// Code generated by ping-terraform-plugin-framework-generator
+
+package captchaproviderssettings_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/acctest"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/provider"
+)
+
+func TestAccCaptchaProviderSettings_BeforeAfter(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() { acctest.ConfigurationPreCheck(t) },
+		ProtoV6ProviderFactories: map[string]func() (tfprotov6.ProviderServer, error){
+			"pingfederate": providerserver.NewProtocol6WithError(provider.NewTestProvider()),
+		},
+		Steps: []resource.TestStep{
+			{
+				// Set with the initial model
+				Config: captchaProviderSettings_BeforeHCL(),
+			},
+			{
+				// Flip to the updated model
+				Config: captchaProviderSettings_AfterHCL(),
+			},
+			{
```

Comment:
If you want it's probably fine to even delete this third step, since flipping again isn't really testing anything terraform-specific. Up to you.

Reply:
putting it back the way it was in step 3 just in case we end up with a dependency test somewhere needing the original default in place for some odd reason